src/main/scala/edu/ie3/simona/model/thermal/ThermalGrid.scala

```diff
@@ -626,10 +659,11 @@ object ThermalGrid {
       possible + rhs.possible,
     )
 
-    def hasRequiredDemand: Boolean = required > zeroMWh
+    def hasRequiredDemand: Boolean = required > zeroKWh
```

Comment:
Was this change necessary?

Reply:
No but we changed maybe MWh to kWh the last year. I would align this here.